examples/hello-grudge.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pyopencl as cl
+from grudge.discretization import DiscretizationCollection
+import grudge.op as op
+from meshmode.mesh.generation import generate_box_mesh
+from meshmode.array_context import PyOpenCLArrayContext
+from arraycontext import freeze, thaw
+from grudge.dof_desc import DTAG_BOUNDARY
+from meshmode.discretization.connection import FACE_RESTR_INTERIOR
```

Comment:
```suggestion
from grudge.dof_desc import DTAG_BOUNDARY, FACE_RESTR_INTERIOR
```
It doesn't seem documented, but `FACE_RESTR_INTERIOR` is exported in `dof_desc` too.